apex-squid/src/test/java/org/fundacionjala/enforce/sonarqube/apex/parser/grammar/ApexGrammarDmlOperationTest.java

```diff
@@ -23,39 +23,35 @@
  */
 package org.fundacionjala.enforce.sonarqube.apex.parser.grammar;
 
+import org.fundacionjala.enforce.sonarqube.apex.parser.ApexRuleTest;
 import org.junit.Before;
 import org.junit.Test;
-
-import org.fundacionjala.enforce.sonarqube.apex.parser.ApexRuleTest;
-
 import static org.sonar.sslr.tests.Assertions.assertThat;
+import static org.fundacionjala.enforce.sonarqube.apex.api.grammar.ApexGrammarRuleKey.DML_OPERATION;
 
-import static org.fundacionjala.enforce.sonarqube.apex.api.grammar.ApexGrammarRuleKey.DML_STATEMENT;
-
-public class ApexGrammarDmlStamentTest extends ApexRuleTest {
-
-    @Before
+public class ApexGrammarDmlOperationTest extends ApexRuleTest{
+@Before
     public void init() {
-        setRootRule(DML_STATEMENT);
+        setRootRule(DML_OPERATION);
     }
 
     @Test
-    public void RulesDmlStament() {
+    public void RulesDmlOperation() {
```

Comment:
It's missing
Upsert Statement
Merge Statement

https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/apex_dml_section.htm